best-practices/javascript.md

```diff
@@ -1,18 +1,12 @@
-# General Javascript Best Practices
+# General JavaScript Best Practices
 
 ## Purpose of this document
 
 - Make everyone aware of and discourage general antipatterns.
 - Make everyone aware of general patterns.
-- Provide a point of reference for what good code looks like at Hx.
+- Provide a point of reference for what good code looks like at Holiday Extras.
 - Get people writing code in a similar way.
 
-Please also see the respective serverside and clientside best practice docs.
```

Comment:
This doesn't link to anything and doesn't make a lot of sense in this context. I can't see any docs that might correspond to this so I'm getting rid of it 🔥